app/build.gradle.kts

```diff
@@ -37,11 +37,11 @@ android {
                 getDefaultProguardFile("proguard-android.txt"),
                 "proguard-rules.pro"
             )
-            resValue("string", "app_name", "Catima")
+
         }
         debug {
             applicationIdSuffix = ".debug"
-            resValue("string", "app_name", "Catima Debug")
```

Comment:
I definitely need the Debug name in the Debug builds so I can tell apart my regular and Debug Catima installs while testing. Please revert this.